models/models.js

```diff
@@ -96,3 +97,15 @@ exports.fetchCommentsById = async (article_id) => {
   );
   return comments.rows;
 };
+
+exports.insertComment = async (newComment, article_id) => {
+  const { username, body } = newComment;
+  const currentTime = Date.now()
+  const date = convertTimestampToDate({created_at:currentTime})
+
+  const comment = await db.query(
+    `INSERT INTO comments (author,body,article_id,votes,created_at) VALUES ($1,$2,$3,$4,$5) RETURNING *;`,
+    [username, body, article_id, 0, date.currentTime]
```

Comment:
Take a look at your seed function - you may find the votes and created_at are actually handled for us!